azure-common-resources/variables.tf

```diff
@@ -287,3 +281,28 @@ variable "platform_client_id" {
 variable "platform_client_secret" {
   type = string
 }
+
+variable "vnet_new" {
+  type    = string
+  default = "new"
+}
```

Comment:
This var should be a boolean imo, for example:
```
variable "create_vnet" {
  type    = bool
  default = true
}
```

Reply:
this is because of the (string) arm output variable. There are two values, "new" or "existing".